aeron-archive/src/main/java/io/aeron/archive/Archive.java

```diff
@@ -369,6 +381,38 @@ public static Archive launch(final Context ctx)
          */
         public static final int ERROR_BUFFER_LENGTH_DEFAULT = 1024 * 1024;
 
+        /**
+         * Should the CRC be computed for each recorder fragment.
+         */
+        public static final String RECORDING_CRC_ENABLED_PROP_NAME = "aeron.archive.recording.crc.enabled";
+
+        /**
+         * Do not compute the CRC for each recorded fragment by default.
+         */
+        public static final boolean RECORDING_CRC_ENABLED_DEFAULT = false;
+
+        /**
+         * Property that specifies fully qualified class name of the {@link io.aeron.archive.checksum.Checksum}
+         * to be used for CRC checksum computation during recording.
+         */
+        public static final String RECORDING_CRC_CHECKSUM_PROP_NAME = "aeron.archive.recording.crc.checksum";
```

Comment:
Pairs should be recording and replaying, or record and replay. Action vs activity. We should pick one or the the other.

`aeron.archive.recording.checksum` and `aeron.archive.replaying.checksum`, or `aeron.archive.record.checksum` and aeron.archive.replay.checksum`. Either can work and stays abstract.  Simply setting either can be used to indicate if enabled or not, no need for a separate property.

Reply:
I went with `aeron.archive.record.checksum` and `aeron.archive.replay.checksum`.